Modules/Clients/Feature/Modules/ClientsExportImportTest.php

```
@@ -16,256 +19,159 @@ class ClientsExportImportTest extends AbstractCompanyPanelTestCase
 
     #[Test]
     #[Group('export')]
-    public function export_contacts_downloads_csv_with_correct_data(): void
+    public function it_dispatches_csv_export_job_v2(): void
     {
         /* Arrange */
-        $contacts = Contact::factory()->for($this->company)->count(3)->create();
+        Queue::fake();
+        Storage::fake('local');
+        $clients = Client::factory()->for($this->company)->count(3)->create();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$clients'. (undefined)

(UnusedLocalVariable)